docs/installation.md

```diff
@@ -17,6 +17,14 @@ On other platforms, install `jrnl` using [Python](https://www.python.org/) 3.6+
 pipx install jrnl
 ```
 
+!!! note
+     `pipx` should be installed through `pip`. Missing dependencies and other issues
```

Comment:
Can we note here that pipx's favored install methods are brew and pip (in that order)? Maybe adding another link to pipx's installation page would be a good idea (in case their installation methods change).